daisy_integration_tests/daisy_e2e.test.gotmpl

```diff
@@ -53,9 +53,6 @@
     "debian9 translate": {
       "Path": "./debian_9_translate.wf.json"
     },
-    "Enterprise Linux - Don't use proxy": {
-      "Path": "./el_import_dont_use_proxy.wf.json"
-    },
```

Comment:
Removing Daisy translate tests means we don't support translate using native Daisy anymore. I think there might sitll be customers who are using translate workflows using Daisy so there's value in keeping these tests in addition to wrapper tests.

Reply:
Oh my, you're right, we have docs on this: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/docs/image-import.md

Based on that, here's my intuition to fix the current timeout problem:
- Remove the new wrapper test
- Restore the old wf.json test
- Update the timeout in it to something larger

Thoughts?

Reply:
I'm ok with points 2 and 3. Regarding the removal of the new wrapper test, if it covers more than the original translate test and if this coverage will not be replicated in other wrapper tests you will add in the future, we can consider keeping it.

Reply:
OK! If it's alright, my intuition is to remove the new test, and only adjust the timeout parameter in the `wf.json` file, since that's the root cause of the issue that we're trying to fix here.